### PR TITLE
Excaption compleated

### LINKED ISSUE #108: Solve problems in SessionManger

### PR DESCRIPTION
**Pull request by Fayzulloabdullayev09**

### CODE REVIEW

File: TheConversationsBot/Domain/Models/User.cs
Comment (Zuxriddin-Zarifov) [inline TheConversationsBot/Domain/Models/User.cs:1]: nimaga user modelga o'zgartirish kiritilgan ?

File: TheConversationsBot/Program.cs
Comment (Zuxriddin-Zarifov) [inline TheConversationsBot/Program.cs:1]: testni ham jo'natish shart emas edi

**Comment by Nodir0506** [comment]
Ustoz Bulardi tushuntirmasangzi, hozir buni qabul qilsan prayektimiz o`zgarib ketadi

File: TheConversationsBot/Program.cs
Comment (Nodir0506) [inline TheConversationsBot/Program.cs:1]: USTOZ kelsangiz buni birga ko`ramiz

**Review by Fayzulloabdullayev09 — COMMENTED** [review]
.

**Comment by Zuxriddin-Zarifov** [comment]
bunda conflig ham bor ekan

**Comment by Fayzulloabdullayev09** [comment]
g

**Pull request closed without merging**